content/release_notes_chefdk.md

```diff
@@ -10,7 +10,7 @@ product = []
 
 This page documents the ChefDK major changes for each release. For
 a detailed list of changes, see the [ChefDK Changelog on
-GitHub](https://github.com/chef/chef-dk/blob/master/CHANGELOG.md)
+GitHub](https://github.com/chef/chef-dk/blob/main/CHANGELOG.md)
```

Comment:
Confirm that we renamed in this repo, since the product is EOL.

Reply:
If `chef-dk` is in the boneyard, then it is probably still `master`. Follow https://github.com/chef/chef-dk/ to confirm